ITR/data/base_providers.py

```diff
@@ -296,7 +311,7 @@ def _calculate_target_projections(self, production_bm: BaseProviderProductionBen
                         self.column_config.REGION: [c.region],
                     }, index=[0])
                     bm_production_data = (
-                            production_bm.get_company_projected_production(company_sector_region_info)
+                            production_bm.get_company_projected_production(company_sector_region_info, scope)
```

Comment:
Again, don't need or want scope associated with production.

Reply:
Yes, agree. We will track this discussion in a different comment